docs/modules/record-encryption/hashicorp-vault/con-vault-setup.adoc

```diff
@@ -52,18 +52,16 @@ filter to be kept separate from any keys used by other systems.  This document a
 will be to prefix the keys used by the filter with the word `KEK_`.  If a different naming convention is used, adapt
 the instructions accordingly.
 
-== Administrator Actor
+== Role of the Administrator
 
-To use the filter, there must be an administrative actor established.  This actor, which is likely to be a human,
-has the responsibility to create keys within Vault for use by the filter.
+In order to use the filter, there must be an *Administrator*.  The Administrator has the responsibility
+to create encryption keys within Vault which will be used to encrypt the records. It is up to the organization
+deploying Record Encryption to decide how the role of the *Administrator* maps into their organization structure.
 
-The Administrator must have permissions to log in to Vault and create keys beneath `transit/keys/KEK_*` in the
+The *Administrator* must have permissions to log in to Vault and create keys beneath `transit/keys/KEK_*` in the
```

Comment:
We document the need for the `transit` engine elsewhere right? 

I was about to suggest that the vault administrator needed the right to enable the transit engine, but realise now that that might be a separate operator to the Key administrator and its worth keeping the two things separate.

Reply:
We have a section "Enable the Transit Engine" on `con-vault-setup.adoc`. 

> but realise now that that might be a separate operator to the Key administrator and its worth keeping the two things separate.

agreed, there's likely to be a separation of concern in an organisation of any size/maturity.